files/pt-br/learn/common_questions/how_does_the_internet_work/index.md

```diff
@@ -0,0 +1,117 @@
+
+> **Nota:** Até o final deste artigo nós estaremos falando apenas a respeito de cabos físicos, mas redes sem fio funcionam da mesma forma
+
+![Dois computadores conectados](https://mdn.mozillademos.org/files/8441/internet-schema-1.png)
+
+Uma rede não é limitada a dois computadores. Você pode conectar quantos computadores desejar. Mas isto se torna complicado. Se você está tentando conectar, digamos, dez computadores, você irá precisar de 45 cabos, com 9 conexões por computador!
+
+![Dez computadores conectados entre si](https://mdn.mozillademos.org/files/8443/internet-schema-2.png)
+
+Para resolver este problema, cada computador na rede está conectado à um pequeno computador especial chamado de _roteador_. Este _roteador_ tem um único trabalho: como um sinalizador em uma estação de trem, ter certeza de que a mensagem enviada por um determinado computador chegue ao computador destinatário corretamente. Para enviar uma mensagem para o computador B, o computador A deve enviar a mensagem para o roteador, que por sua vez encaminha a mensagem para o computador B e tem a certeza de que a mensagem não foi entregue ao computador C.
+
+Uma vez que nós adicionamos um roteador no sistema, nossa rede de 10 computadores apenas necessitará de 10 cabos: uma unica conexão para cada computador e um roteador com 10 conexões.
+
+![10 computadores com um roteador](https://mdn.mozillademos.org/files/8445/internet-schema-3.png)
+
+### Uma rede de redes
+
+Por enquanto, tudo bem. Mas como conectar centenas, milhares, bilhões de computadores? Claro que um unico _roteador_ não pode se adaptar para tanto, mas, se você ler com cuidado, nós dissemos que um _roteador_ é um computador como qualquer outro, então o que nos impede de conectar dois _roteadores_ juntos? Nada, então façamos isto.
+
+![Dois roteadores conectados](https://mdn.mozillademos.org/files/8447/internet-schema-4.png)
+
+Conectando computadores a roteadores, e então roteadores a roteadores nós podemos escalar nossa rede infinitamente.
+
+![Roteadores conectados a roteadores](https://mdn.mozillademos.org/files/8449/internet-schema-5.png)
+
+Esta rede é muito parecida com o que chamamos de Internet, mas alguma coisa está faltando. Nós contruímos tais redes para nossos próprios fins. Existem outras redes além das nossas ligadas em outros lugares: nossos amigos, vizinhos, qualquer pessoa que tenha uma rede de computadores. Mas é inviável ligarmos cabos entre nossas casas e o resto do mundo, então como nos podemos lidar com isso? Muito bem, já existem cabos ligados a sua casa, como por exemplo, cabos de eletricidade e telefone. A estrutura do telefone já conecta nossa casa com o resto do mundo. Para conectar nossa rede a rede telefonica, precisamos de um equipamento especial chamado _modem_. Este *modem* transforma a informação da nossa rede em uma informação gerenciável pela rede telefônica e vice-versa.
```

Comment:
```suggestion
Esta rede é muito parecida com o que chamamos de Internet, mas alguma coisa está faltando. Nós construímos tais redes para nossos próprios fins. Existem outras redes além das nossas ligadas em outros lugares: nossos amigos, vizinhos, qualquer pessoa pode ter uma rede de computadores. Mas é inviável ligarmos cabos entre nossas casas e o resto do mundo, então como nos podemos lidar com isso? Muito bem, já existem cabos ligados a sua casa, como por exemplo, cabos de eletricidade e telefone. A estrutura do telefone já conecta nossa casa com o resto do mundo, então é exatamente o que nós precisamos. Para conectar nossa rede a rede telefônica, precisamos de um equipamento especial chamado _modem_. Este *modem* transforma a informação da nossa rede em uma informação gerenciável pela rede telefônica e vice-versa.
```

Reply:
Corrigindo 2 erros de digitação "contruímos" estava faltando o S e "telefonica" estava sem acento.
Também troquei "que tenha" por "pode ter" considerando a versão em inglês "anyone can have their own network".
A adição da frase "então é exatamente o que nós precisamos" foi feita seguindo também a versão em inglês "so it is the perfect wire we need".